backend/src/main/resources/static/semantic-hub-openapi.yaml

```diff
@@ -116,6 +116,50 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /models/lookup:
```

Comment:
Having this endpoint under /models is not ideal because there is also /models/{urn} but lookup is not a valid model urn anyway. 
In addition, this API might change at some point, e.g., when aligning it further with AAS activities.